src/main/resources/lang-type-mapping.properties

```diff
@@ -16,6 +16,7 @@ JS.JSON_OBJECT=object
 JS.JSON_ARRAY=Array
 JS.API=api
 JS.OBJECT=ALL
+JS.THROWABLE=object
```

Comment:
there is actually an object Error in javascript (but it's an object - so ignore this comment)